src/profiles.js

```diff
@@ -0,0 +1,77 @@
+import tv4 from 'tv4'
+import jsonlint from 'json-lint'
+import _ from 'lodash'
+
+import Registry from './registry'
+
+class Profiles {
+
+  constructor(remote = false) {
+    this._registry = new Registry(remote)
+    this.remote = remote
```

Comment:
If we need it as public API to please use patter `this._remote = remote` + getter method. It helps to document class API.